easybuild/easyconfigs/c/Cube/Cube-20190321-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,42 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2019 Juelich Supercomputing Centre, Germany
+# Authors::   Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+easyblock = 'Bundle'
+
+name = 'Cube'
+version = '20190321'
```

Comment:
There is no "main" component.  The former `Cube` package has been split into three disjoint componts: `CubeWriter`, `CubeLib`, and `CubeGUI`.  They now evolve somewhat independently and version numbers may diverge more over time.  In this case, the three components have been released on the same date (chosen as `version`), but this doesn't have to be the case in the future. Dependent packages may require only some of them or in different use cases (`dependency` vs. `builddependency`, see the `Score-P` easyconfig), thus I would really prefer to keep them separate.

In principle, the `Cube` bundle isn't required at all -- I've just added it as a convenience for people who are used to the old `Cube` package.  While we also provide a convenience bundle tarball including all components on our web page, its version is a simple count that has very little to do with the versions included (as the versions are allowed to diverge, it is hard to come up with a common version).  Having said all this, I'm also OK with removing the bundle if you think it causes too much confusion. (This would also magically "fix" the failing Travis check :wink:)